conferences/types.go

```diff
@@ -47,3 +47,165 @@ type ConferenceSlot struct {
 	// issue sponsor tickets and those cannot be bought individually)
 	AvailableToPublic bool
 }
+
+// ClaimPayment represents a payment for N claims
+type ClaimPayment struct {
+	ID uint64 `gaum:"field_name:id"`
+	// ClaimsPayed would be what in a bill one see as detail.
+	ClaimsPayed []*SlotClaim
+	Payment     []FinancialInstrument
```

Comment:
`FinancialInstrument` is an interface so it will be in fact filled with pointers to concrete types, SlotClaims is not and that is Why I used the pointers. I am willing to change the later to concretes.